src/resources/js/v2/rsvp-block.js

```diff
@@ -138,6 +135,35 @@ tribe.tickets.rsvp.block = {};
 		} );
 	};
 
+	/**
+	 * Handle the RSVP toggle for listing in public attendee list.
+	 *
+	 * @since TBD
+	 *
+	 * @param {event} e submission event
+	 */
+	obj.handleDisplayToggle = function( e ) {
+		e.preventDefault();
+
+		const $input = $( e.target );
+		const rsvpId = $input.data( 'rsvp-id' );
+		const checked = $input.prop( 'checked' );
+		const attendeeIds = $input.data( 'attendee-ids' );
+		const nonce = $input.data( 'opt-in-nonce' );
+		const $container = e.data.container;
+
+		var data = {
+			action: 'tribe_tickets_rsvp_handle',
+			ticket_id: rsvpId,
+			step: 'opt-in',
+			opt_in: checked,
+			opt_in_none: nonce,
```

Comment:
`opt_in_none`? Is this how it is handled in the PHP or is this a typo on the JS side?

Reply:
No, that's a typo haha🤦‍♂️